cvat/apps/webhooks/event_type.py

```diff
@@ -0,0 +1,56 @@
+from .models import WebhookTypeChoice
+
+def event_name(action, resource):
+    return f"{action}:{resource}"
+
+class Events:
+    RESOURCES = {
+        "project":      ["create", "update", "delete"],
+        "task":         ["create", "update", "delete"],
+        "issue":        ["create", "delete"], # TO-DO: implement issue_updated
```

Comment:
Can we add the issue updated event now? What is a reason for the TODO?

Reply:
Yes, it's mistake, `update:issue` event is supported fine. Fixed it.

I wanted to mention here that we don't support `update:invitation`, because 1)  Updating invitation differs from updating others models in CVAT 2) I think that PATCH invitation it is not popular use case